cds/modules/fixtures/data/categories.json

```diff
@@ -37,7 +38,8 @@
     "types": ["VIDEO"],
     "access": {
       "public": true,
-      "restricted": []
+      "restricted": [],
```

Comment:
do we want to set any default e-group (the same one?) if they change the video to restricted, or keep it empty?

Reply:
I doubt that we can assume that the same egroup should apply to restrict videos